### ISSUE TITLE
Dataset and train test split for pretrained model

### ISSUE THREAD

**Issue by twoleggedeye** [lmnt-com/wavegrad#6]
Hi!
What dataset did you use for training the 24khz model? Also, what train/test split did you use for training?

I am having a poor  samples quality on LJSpeech inference with the model (I resampled the dataset beforehand), so I assume this model was not trained on it?

**Comment by sharvil** [comment]
The 24kHz model was trained on the VCTK corpus with a random (but stable) 87.5% train/12.5% test split. The [audio samples](https://lmnt.com/assets/wavegrad/24kHz/) have a few examples from the test set. You may also be interested in [DiffWave](https://github.com/lmnt-com/diffwave) which follows a similar architecture as WaveGrad but, in my experience, produces better results.

**Comment by twoleggedeye** [comment]
Great, thank you!

**Comment by sharvil** [comment]
I can provide the procedure for generating the train/test split:

Consider each `(text_file, wav_file)` pair as a record.
For each record, read the text file as a UTF-8 string.
Remove leading and trailing whitespace from the text.
Generate a SHA1 hash of the text, and consider the 3 least-significant bits of the most-significant byte of the hash.
If they're all 0, this record belongs in the test set; otherwise it belongs in the train set.
